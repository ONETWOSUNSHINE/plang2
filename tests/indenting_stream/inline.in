foo\i \a bar\nbaz
quux    foobar\b
corge
