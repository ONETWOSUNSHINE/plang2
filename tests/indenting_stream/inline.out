foo bar baz quux foobar
    corge
